src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -117,7 +117,7 @@ func doMain() error {
 
 	parseFlags()
 
-	cfg, err := config.Init()
+	cfg, err := config.Init(packageVersion)
```

Comment:
Hello @atavism!

`packageVersion` was initially intended for autoupdates only. If no `VERSION` is passed to the Makefile it will use the git revision and when compiled in development mode it will use `9999.99.99`. I think the git revision would be more informative but I also think it would be harder to  associate it with an actual release, as we'll always have to lookup the git revision.